tests/distributions/test_rejector.py

```diff
@@ -149,8 +149,8 @@ def test_shape_augmented_beta(alpha, beta):
     z = dist.rsample()
     cost = z.sum()
     (cost + cost.detach() * dist.score_parts(z)[1]).backward()
-    mean_alpha_grad = alphas.grad.data.mean()
-    mean_beta_grad = betas.grad.data.mean()
+    mean_alpha_grad = alphas.grad.data.mean().item()
```

Comment:
Why not simply `alphas.grad.mean().item()`?

Reply:
Yup, in this case we can just use that. Will change here and other patterns that I see.